autogpts/autogpt/pyproject.toml

```diff
@@ -73,6 +73,10 @@ agbenchmark = { path = "../../benchmark", optional = true }
 google-cloud-logging = "^3.8.0"
 google-cloud-storage = "^2.13.0"
 psycopg2-binary = "^2.9.9"
+nbformat = "^5.10.3"
+nbconvert = "^7.16.3"
+nbclient = "^0.10.0"
+ipykernel = "^6.29.4"
```

Comment:
Please insert these in their alphabetical positions in the primary dependency list (L24-65)